src/host/js/smooch.js

```diff
@@ -135,9 +135,11 @@ function onSmoochReady(_Lib) {
 
 function injectFrame() {
     if (!iframe) {
+        let loaded = false;
         iframe = document.createElement('iframe');
+
+        iframe.frameborder = 0;
```

Comment:
This is just to prevent the border to show up until the stylesheet is applied. I